src/lib.rs

```diff
@@ -71,17 +71,17 @@ use std::io::{ErrorKind::NotFound, Write};
 use std::path::{Path, PathBuf};
 use thiserror::Error;
 
-#[cfg(not(any(feature = "toml_conf", feature = "yaml_conf")))]
+#[cfg(not(any(feature = "toml_conf", feature = "yaml_conf", feature = "ron_conf")))]
 compile_error!(
     "Exactly one config language feature must be enabled to use \
-confy.  Please enable one of either the `toml_conf` or `yaml_conf` \
-features."
+confy.  Please enable one of either the `toml_conf`, `yaml_conf` \
+or `ron_conf` features."
 );
 
-#[cfg(all(feature = "toml_conf", feature = "yaml_conf"))]
+#[cfg(all(feature = "toml_conf", feature = "yaml_conf", feature = "ron_conf"))]
```

Comment:
I think we can manage this with:
```rust
#[cfg(any(
    all(feature = "toml_conf", feature = "yaml_conf"),
    all(feature = "toml_conf", feature = "ron_conf"),
    all(feature = "ron_conf", feature = "yaml_conf"),
))]
```
I haven't tested this locally yet as I'm not at home but shouldn't that work for covering all cases?

Reply:
Resolved in https://github.com/rust-cli/confy/pull/71/commits/bfda06254f73eca01e9ca908c1d7966cbdc016b2. I decided to go for the above solution. This might become verbose quickly if more features were to be added but I don't think it's too big of an issue right now.